src/main/java/io/jenkins/plugins/models/JiraIssues.java

```diff
@@ -0,0 +1,23 @@
+package io.jenkins.plugins.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class JiraIssues {
+    @JsonProperty("issues")
+    public List<JiraIssue> issues = new ArrayList();
+
+    @JsonProperty("total")
+    public int getTotal() {
+      return this.issues.size();
+    }
+
+    public JiraIssues() {
```

Comment:
constructors go above methods in java ordering normally, not sure why you even need a constructor here though it should be implied

https://stackoverflow.com/a/10264498/4951015